ci/build_wheel.sh

```diff
@@ -6,9 +6,18 @@ set -euo pipefail
 rapids-logger "Install CUDA Toolkit"
 source "$(dirname "$0")/install_latest_cuda_toolkit.sh"
 
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+
+# This is the version of the suffix with a preceding hyphen. It's used
+# everywhere except in the final wheel name.
+PACKAGE_CUDA_SUFFIX="-${RAPIDS_PY_CUDA_SUFFIX}"
+
+# Patch project metadata files to include the CUDA version suffix.
+sed -i "s/^name = \"pynvjitlink\"/name = \"pynvjitlink${PACKAGE_CUDA_SUFFIX}\"/g" pyproject.toml
+
 rapids-logger "Build wheel"
-mkdir -p ./wheel-build
-pip wheel . --wheel-dir=./wheel-build -vvv
+mkdir -p ./dist
+python -m pip wheel . --wheel-dir=./dist -vvv --disable-pip-version-check
```

Comment:
This change is mostly for future-proofing since pynvjitlink doesn't have any dependencies right now.
```suggestion
python -m pip wheel . --wheel-dir=./dist -vvv --disable-pip-version-check --no-deps
```

Reply:
I don't know what this flag is meant to do. Is this command doing an isolated build? Does it skip installing the "build system requirements" when this is specified?

Reply:
It skips run requirements. `pip wheel` downloads runtime dependencies by default. The build is isolated by default, yes. There is of course no way to turn off downloading build dependencies when doing an isolated build since that would always be broken.